### PR TITLE
html page of the user page in ionic app from juggling-app

### PR DESCRIPTION
**Pull request by LauriFernandez**
creation of a web page to make an ionic application.
This page is using to display the profile user and update it.

### CODE REVIEW

File: laurieNewFile.txt [hunk 1]
```diff
@@ -1 +1,39 @@
...
+<ion-header>
+
+    <ion-navbar>
+        <ion-title>UserProfilePage</ion-title>
+    </ion-navbar>
+
+</ion-header>
+
+
+<ion-content padding>
```
Comment (artmoni) [inline laurieNewFile.txt:16]: pourquoi utilises tu `padding` ?

File: laurieNewFile.txt [hunk 2]
```diff
@@ -1 +1,39 @@
-new file for the git formation !! 
+<!--
+  Generated template for the UserProfilePage page.
+
+  See http://ionicframework.com/docs/components/#navigation for more info on
```
Comment (juliagerol) [inline laurieNewFile.txt:4]: why this function is using this var?

File: laurieNewFile.txt [hunk 3]
```diff
@@ -1 +1,39 @@
...
+            <ion-item>
+                <ion-label color="primary">Mail :</ion-label>
+                <ion-input placeholder="paul@gmail.com" [(ngModel)]="user.mail" type="mail"></ion-input>
+            </ion-item>
+            <ion-item>
+                <ion-label color="primary">Numéro de téléphone :</ion-label>
+                <ion-input placeholder="0612345678" [(ngModel)]="user.tel" type="tel"></ion-input>
+            </ion-item>
+            <ion-footer>
+                <button ion-button large center (click)="update()">Mettre à jour mon profil</button>
```
Comment (artmoni) [inline laurieNewFile.txt:33]: c'est super d'utiliser le `ion-button`

File: laurieNewFile.txt [hunk 4]
```diff
@@ -1 +1,39 @@
...
+
+    <ion-navbar>
+        <ion-title>UserProfilePage</ion-title>
+    </ion-navbar>
+
+</ion-header>
+
+
+<ion-content padding>
+    <div *ngIf="user">
```
Comment (LauriFernandez) [inline laurieNewFile.txt:17]: why you need to verify the user here ?

**Review by juliagerol — COMMENTED** [review]
i think it's cool!

**Review by artmoni — COMMENTED** [review]
merci ca a l'air pas mal

**Pull request merged**